artifact.sh

```diff
@@ -49,6 +49,9 @@ fi
 
 bash build.sh
 
+# CloudFront Lambda edge tests:
+npm run test-cloudfront
+
```

Comment:
Bellow (line 59) we should also prepare the folder that will be uploaded on the cloudfront S3 bucket.
Something like:  
```
rsync server/dist cloudfront/distrib --exclude "index.html, config*.js, cloudfront*.js"
cp server/dist/cloudfront*.js cloudfront/lambda
```

Then later in the publish process we can push the website content to cloudfront S3 and pulish the lambda

Reply:
Good point.
Too late, it's merged, we will do another PR.